docs/account-validation.md

```diff
@@ -0,0 +1,295 @@
+# Account Validation
+
+Moov Customers project supports following strategies and vendors to validate accounts:
+
+* micro-deposits validation with Moov [PayGate](https://github.com/moov-io/paygate)
+* instant validation with [Plaid](https://plaid.com/products/auth/)
```

Comment:
I think the Plaid / Mx items here should link to their sections down below. Those sections can link off to plaid.com / mx.com